make install does not work

`make install` does not work because of change of directories structure (in build directory). I received following error using `make install`:
```
cp build/bin/snapd /usr/local/bin/
cp: cannot stat 'build/bin/snapd': No such file or directory
Makefile:38: recipe for target 'install' failed
make: *** [install] Error 1
```
Current tree of build directory:
```
build
│   ├── darwin
│   │   └── x86_64
│   │       ├── snapctl
│   │       ├── snapd
│   │       ├── snap-plugin-collector-anothermock1
│   │       ├── snap-plugin-collector-mock1
│   │       ├── snap-plugin-collector-mock2
│   │       ├── snap-plugin-collector-mock2-grpc
│   │       ├── snap-plugin-processor-passthru
│   │       ├── snap-plugin-processor-passthru-grpc
│   │       ├── snap-plugin-publisher-mock-file
│   │       └── snap-plugin-publisher-mock-file-grpc
│   └── linux
│       └── x86_64
│           ├── snapctl
│           ├── snapd
│           ├── snap-plugin-collector-anothermock1
│           ├── snap-plugin-collector-mock1
│           ├── snap-plugin-collector-mock2
│           ├── snap-plugin-collector-mock2-grpc
│           ├── snap-plugin-processor-passthru
│           ├── snap-plugin-processor-passthru-grpc
│           ├── snap-plugin-publisher-mock-file
│           └── snap-plugin-publisher-mock-file-grpc
```
Earlier tree of build directory:
```
 build
│   ├── bin
│   │   ├── snapctl
│   │   └── snapd
│   └── plugin
│       ├── snap-plugin-collector-anothermock1
│       ├── snap-plugin-collector-mock1
│       ├── snap-plugin-collector-mock2
│       ├── snap-plugin-collector-mock2-grpc
│       ├── snap-plugin-processor-passthru
│       ├── snap-plugin-processor-passthru-grpc
│       ├── snap-plugin-publisher-mock-file
│       └── snap-plugin-publisher-mock-file-grpc
```

--- a/Makefile
+++ b/Makefile
@@ -15,6 +15,9 @@
 #See the License for the specific language governing permissions and
 #limitations under the License.
 
+OS = $(shell uname -s | tr '[:upper:]' '[:lower:]')
+ARCH = $(shell uname -m)
+
 default:
 	$(MAKE) deps
 	$(MAKE) all
@@ -35,7 +38,7 @@ check:
 all:
 	bash -c "./scripts/build_snap.sh"
 install:
-	cp build/bin/snapd /usr/local/bin/
-	cp build/bin/snapctl /usr/local/bin/
+	cp build/$(OS)/$(ARCH)/snapd /usr/local/bin/
+	cp build/$(OS)/$(ARCH)/snapctl /usr/local/bin/
 proto:
 	cd `echo $(GOPATH) | cut -d: -f 1`; bash -c "./src/github.com/intelsdi-x/snap/scripts/gen-proto.sh"

